include/opt-sched/Scheduler/lnkd_lst.h

```diff
@@ -555,6 +560,7 @@ template <class T> inline T *LinkedList<T>::GetNxtElmnt() {
 }
 
 template <class T> inline T *LinkedList<T>::GetPrevElmnt() {
+  assert(rtrvEntry_ != NULL);
```

Comment:
Asserts in

`PriorityList<T, K>::GetNxtPriorityElmnt`
`PriorityList<T, K>::ViewNxtPriorityElmnt`

should also be fixed since `LinkedList<T>::rtrvEntry_` may by NULL if `wasTopRmvd_`